BaseApi.js

```diff
@@ -68,8 +68,14 @@ const makeGetRequest = (url, token = null) => {
     method: 'GET',
     headers: getHeaders(token)
   })
-    .then(response => response.json())
-    .then(unpackedData => unpackedData)
+    .then(response => {
+      const statusCode = response.status
+      const data = response.json()
+      return Promise.all([ statusCode, data ])
+    })
+    .then(values => {
+      return {statusCode: values[0], data: values[1]}
```

Comment:
the tests for `Idea.js` now fail because the mocked Api still returns a single value, not the two like here

Reply:
yeah, I know, I wrote in the PR description that I need help fixing the tests.. :see_no_evil: